parser/fuzz/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+  name    = "ezno-parser-fuzz"
+  version = "0.0.0"
+  publish = false
+  edition = "2021"
+
+  [package.metadata]
+    cargo-fuzz = true
+
+[dependencies]
+  libfuzzer-sys     = "0.4"
+  pretty_assertions = "1.3.0"
+
+  [dependencies.ezno-parser]
+    path = ".."
+
+# Prevent this from interfering with workspaces
+[workspace]
+  members = ["."]
```

Comment:
This line makes me believe it shouldn't be added to the larger workspace.

If only rust-analyzer was smarter 😭